inference-engine/src/transformations/include/ngraph_ops/framework_node.hpp

```diff
@@ -10,30 +10,35 @@
 
 
 #include "ngraph/op/op.hpp"
+#include "ngraph/strides.hpp"
 #include "ngraph/partial_shape.hpp"
 
 namespace ngraph {
 namespace op {
 
-class TRANSFORMATIONS_API FrameworkNodeAttrs {
+class FrameworkNodeAttrs: public std::unordered_map<std::string, std::string> {
```

Comment:
Why do you need to have an inheritance from unordered_map?
You get undefined behavior if you attempt to delete an object of your type through a pointer to a `std::unordered_map`, because `unordered_map` doesn't have virtual destructor.

Reply:
I did it to reuse map functionality. But yes, it has bad consequences. So I added necessary methods to FrameworkNodeAttr class.